internal/pkg/skuba/kubernetes/kubelet.go

```diff
@@ -36,7 +37,7 @@ func DisarmKubelet(node *v1.Node) error {
 }
 
 func disarmKubeletJobName(node *v1.Node) string {
-	return fmt.Sprintf("caasp-kubelet-disarm-%s", node.ObjectMeta.Name)
```

Comment:
Can we keep the name but strip it so `caasp-kubelet-disarm-%s` does not exceed 63 char ? Having 41 chars from the hostname is still a good indication of which node it is running on.

Reply:
It will never exceed 63 char, because we don't append the node name anymore. Here `%s` is the random UUID which has 32 chars.

Reply:
yes but what I mean is instead of using a UUID, we strip the hostname,

let's say we have a nodename: `my-node-01-dc-12-a-lot-of-characters-strip-end-so-it-fits` (len =57), we strip it so it gives `caasp-kubelet-disarm-my-node-01-dc-12-a-lot-of-characters-strip` (len=63)

This way the name of the pod still means something.

Reply:
If we are using long hostnames like the ones described I think there are chances that the difference on the hostnames are at the end (the stripped part). What would happen if we strip like this? We can have conflicting job names because the stripped part (the one that univocally refers to a node) was stripped out in both cases.

I would prefer if we go with an UUID (or even, a SHA1 sum or MD5 sum of the node name being removed). This way, it's not a complete random UUID changing among runs, but a consistent transformation of the nodename into a sum, that has a known length.

For checking what node this job refers to you can just describe it and check the `nodeSelector`.

Reply:
@ereslibre I like the idea with SHA1 sum. Will do that.

Reply:
Yes. Which is totally fine, our limit is 63.